$MirOS: ports/net/curl/patches/patch-docs_examples_anyauthput_c,v 1.1 2008/04/05 21:54:38 tg Exp $
--- docs/examples/anyauthput.c.orig	Fri May 23 09:05:31 2008
+++ docs/examples/anyauthput.c	Tue Aug 26 19:03:21 2008
@@ -66,7 +66,7 @@ static size_t read_callback(void *ptr, s
 
   retcode = read(fd, ptr, size * nmemb);
 
-  fprintf(stderr, "*** We read %d bytes from file\n", retcode);
+  fprintf(stderr, "*** We read %zd bytes from file\n", retcode);
 
   return retcode;
 }
